app-text/texlive/additional-files/createSubpackageInfos.pl

```diff
@@ -0,0 +1,98 @@
+#!/bin/perl
+
```

Comment:
Re-adding review comments from #6115

Should I add a license header here?

As to which one, I suppose it would have to be GPL v2 because it depends on such code. But I am open to MIT too, if that is possible.

Reply:
MIT is fine.

The way GPL is "viral" is often misunderstood: it never adds further restrictions on things that are under a more open license and grant more rights.

So, if someone was to download the GPL'ed package, and then copy just some bits from this script that is under MIT license, and use that as allowed by the MIT license, that's fine.

It's not like this makes a big difference for a script anyway, since most of the extra things in GPL is about distributing sourcecode alongside binaries. In this case, there are no binaries because this is a script.